src/go/k8s/pkg/resources/certmanager/pandaproxy_api.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+// Package certmanager contains resources for TLS certificate handling using cert-manager
+// nolint:dupl // this is similar to the admin pki
+package certmanager
+
+import (
+	cmmeta "github.com/jetstack/cert-manager/pkg/apis/meta/v1"
+	"github.com/vectorizedio/redpanda/src/go/k8s/pkg/resources"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const (
+	pandaproxyAPI = "proxy"
```

Comment:
what is this? It does not seem to be used in this commit and from the name I can't really figure it out 🤔

Reply:
It's used under `pki.go`. It follows the same naming as adminAPI, kafkaAPI. We could make all of those variables more descriptive in a separate PR.